reactComponents/src/searchbar/redux/tokenSearchSlice.ts

```diff
@@ -8,16 +8,14 @@ import {TokenSearchState} from "./types";
 
 export const setPair = createAsyncThunk(
   'token/setPair',
-  async ({ selectedPair }:any) => {
-    console.log("setPair")
+  async ({ selectedPair }:any) => {    
```

Comment:
please add a type

Reply:
Regarding backend logic, we still need to improve.

Reply:
unresolving this

Reply:
setPair and resetSearchOnNewExchange are never used. I am not sure why  previous dev defined these methods.
I removed.